src/gitlab.js

```diff
@@ -51,6 +53,7 @@ export default class VerdaccioGitLab implements IPluginAuth {
   config: VerdaccioGitlabConfig;
   authCache: AuthCache;
   logger: Logger;
+  accessLevel: VerdaccioGitlabAccessLevel;
```

Comment:
Allows to specify the minimal role level to get access (like the existing one for publish).